tests/quality_pragmaitc/test_refinement.py

```diff
@@ -24,63 +30,112 @@ def test_shouldBeApplicable():
 
     fullContext.append(contextCurrent)
 
-    #goal.addApplicableContext(contextCurrent)
+    goal.addApplicableContext(contextCurrent)
     task.addApplicableContext(contextCurrent)
     delegation.addApplicableContext(contextCurrent)
 
-    #assert True == goal.isApplicable(fullContext)
-    assert True == task.isApplicable(fullContext)
-    assert True == delegation.isApplicable(fullContext)
+    assert True is goal.isApplicable(fullContext)
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [Use of assert detected. The enclosed code will be removed when compiling to optimised byte code.](https://app.codacy.com/gh/lesunb/p2psimpy/pullRequest?prid=4861978)